finality-aleph/src/network/service.rs

```diff
@@ -555,13 +502,13 @@ mod tests {
         );
 
         let expected_messages = HashSet::from_iter(
-            identities
+            peer_ids
                 .iter()
                 .take(opened_authorities_n)
-                .map(|identity| {
+                .map(|peer_id| {
                     messages
                         .iter()
-                        .map(move |m| (m.clone(), identity.1, Cow::Borrowed(ALEPH_PROTOCOL_NAME)))
+                        .map(move |m| (m.clone(), *peer_id, Protocol::Generic))
```

Comment:
you might be able to get rid of deref and clone by using `into_iter`, But I'm Not a Compiler :slightly_smiling_face: